wifiphisher/extensions/karma.py

```diff
@@ -0,0 +1,108 @@
+"""
+Extension that check victims associated with rogue AP by KARMA attack
+"""
+
+import itertools
+from collections import defaultdict
+
+
+class Karma(object):
+    """
+    Handles for printing KARMA attack information
+    """
+
+    def __init__(self, data):
+        """
+        Setup the class with all the given arguments.
+
+        :param self: A Karma object.
+        :param data: Shared data from main engine
+        :type self: Karma
+        :type data: dictionary
+        :return: None
+        :rtype: None
+        """
+        self._data = data
+        self._packets_to_send = defaultdict(list)
+        self._mac2ssid_dict = defaultdict()
+        self._roundrobin_macssids = itertools.cycle([])
+
+    def get_packet(self, packet):
+        """
+        :param self: A Karma object
+        :param packet: A scapy.layers.RadioTap object
+        :type self: Karma
+        :type packet: scapy.layers.RadioTap
+        :return: empty list
+        :rtype: list
+        """
+        return self._packets_to_send
+
+    def send_output(self):
+        """
+        Send the output the extension manager
+        :param self: A Karma object.
+        :type self: Karma
+        :return: A list with the password checking information
+        :rtype: list
+        ..note: In each packet we ask roguehostapd whether there are victims
+        associated to rogue AP
+        """
+        info = []
+        is_change = False
+        if not self._data.args.force_hostapd:
```

Comment:
This check has already be done and the module wouldn't load if the user was using the regular hostapd. I believe it's safe to remove it.